projects/core/src/configurator/commons/facade/configurator-groups.service.ts

```diff
@@ -339,7 +215,14 @@ export class ConfiguratorGroupsService {
     );
   }
 
-  getPreviousGroupId(owner: GenericConfigurator.Owner): Observable<string> {
+  /**
+   * Returns a previous group ID.
```

Comment:
See above. Better like 'Returns the group ID of the group that is preceeding the current one in a sequential order'